### PR TITLE
Dependencies and goversion

### PR DESCRIPTION
**Pull request by paologallinaharbur**
Just updating dependencies and goversion to fix tests.

The manager after the fix is taking a bit more to initialize, that was breaking tests

### CODE REVIEW

**Review by gsanchezgavier — APPROVED** [review]
thanks for take care of it

File: Dockerfile.dev [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-alpine3.14 as builder
+FROM golang:1.18-alpine3.14 as builder
```
Comment (roobre) [inline Dockerfile.dev:1]:
```suggestion
FROM golang:1.18-alpine3.15 as builder
```

Comment (roobre) [reply Dockerfile.dev:1]: Do we need to make changes to the prod Dockerfile?

**Review by roobre — APPROVED** [review]

**Review by sigilioso — APPROVED** [review]

File: Dockerfile.dev (hunk 1 above)
Comment (paologallinaharbur) [reply Dockerfile.dev:1]: I'll change those as well!

**Pull request merged**